misc/remove-local-blobstore-agent-credentials.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- type: remove
+  path: /instance_groups/name=blobstore/properties/blobstore/agent/user?
```

Comment:
Shouldn't it be `name=bosh` instead of `name=blobstore`. In addition wouldn't it make more sense to put these changes directly into `blobstore-signed-urls.yml`? Since the bionic stemcell supports singed URLs. So there is no use-case for having to support signed URLs and agent credentials at the same time.

Reply:
I've fixed the `name=` issue.

About gathering all changes into `blobstore-signed-urls.yml` at the same time, all right I get it: as `bosh-deployment` does not support Xenial stemcells anymore, we're in a context where signed URLs are definitely supported, so we don't need to support both signed URLs and agent blobstore credentials.
(I'm also referring to that discussion in https://github.com/cloudfoundry/bosh/pull/2327#discussion_r726182426)

All right, I'll merge that in the other ops file, then.